src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({person}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{person.name ? `I am ${person.age}` : null}</p>
```

Comment:
```suggestion
    {person.age && <p className="Person__age">`I am ${person.age}`</p>
```
why do you check the name?